static/app/components/events/contexts/utils.tsx

```diff
@@ -249,6 +295,96 @@ export function getContextMeta(event: Event, type: string): Record<string, any>
   }
 }
 
+export function getFormattedContextData({
```

Comment:
Where is this used? Can't seem to find it with cmd-f

Reply:
Yeah it's not used anywhere yet but will be here https://github.com/getsentry/sentry/pull/68081. I figured I'd add it in this PR because it the reason I needed to make all those other functions exports for each context file.

Reply:
Gotcha, makes sense!